cea/globalvar.py

```diff
@@ -19,13 +19,13 @@
 
 class GlobalVariables(object):
     def __init__(self):
-        self.scenario_reference = r'c:\reference-case-zug\baseline'
+        self.scenario_reference = r'c:\reference-case-open\baseline'
         self.print_partial = 'hourly' # hourly or monthly for the deamnd script
         self.print_yearly = True # print yearly values
         self.print_yearly_peak = True # print peak values
         self.date_start = '2016-01-01'  # format: yyyy-mm-dd
         self.seasonhours = [3216, 6192]
-        self.multiprocessing = True  # use multiprocessing / parallel execution if possible
+        self.multiprocessing = False  # use multiprocessing / parallel execution if possible
```

Comment:
i totally agree. I think we should add it to every script that has multiprocessing enabled. I believe the plot demands has it too...